REQUIRE

```diff
@@ -1,3 +1,7 @@
 julia 0.5-
 CoordinateTransformations 0.3.0
-StaticArrays 0.0.4
+
+# Enforce that:
+# - Julia 0.6 only using StaticArrays 0.5.0 and up
+# - Julia 0.5 uses StaticArrays between 0.0.4 and 0.4.0-
+StaticArrays 0.0.4 0.4.0- 0.5.0
```

Comment:
It's not pretty but should get the job done. If we really want to support 0.4.0 of StaticArrays we could make the conditional check `Pkg.installed("StaticArrays")` versus using `VERSION`. I ended up doing things this way as the `Pkg.installed` calls isn't as performant as `VERSION`